db/migrate/20160113181700_copy_body_string_to_body_text.rb

```diff
@@ -0,0 +1,5 @@
+class CopyBodyStringToBodyText < ActiveRecord::Migration
+  def change
+     Hubstats::Comment.update_all("body_text=body")
```

Comment:
so you should change `change` to be `up` and create a  `down` method that does the reverse.